specification/eventgrid/data-plane/Microsoft.ContainerService/stable/2018-01-01/ContainerService.json

```diff
@@ -27,6 +27,58 @@
           "description": "The highest PATCH Kubernetes version considered preview for the ManagedCluster resource. There might not be any version in preview at the time of publishing the event",
           "type": "string"
         }
+      },
+      "x-ms-examples": {
+        "newKubernetesVersionAvailableCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/new_kubernetes_version_available.json"
+        },
+        "newKubernetesVersionAvailableEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/new_kubernetes_version_available.json"
+        }
+      }
+    },
+    "ContainerServiceClusterOutOfSupportEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ContainerService.ClusterOutOfSupport event",
+      "type": "object",
+      "properties": {
+        "kubernetesVersion": {
+          "description": "The Kubernetes version of the ManagedCluster resource",
+          "type": "string"
+        },
+        "outOfSupport": {
+          "description": "Indicates if the Kubernetes version of the ManagedCluster resource is out of support",
+          "type": "boolean"
+        }
+      },
+      "x-ms-examples": {
+        "clusterOutOfSupportCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/cluster_out_of_support.json"
+        },
+        "clusterOutOfSupportEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/cluster_out_of_support.json"
+        }
+      }
+    },
+    "ContainerServiceNodePoolRollingProgressEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ContainerService.NodePoolRollingProgress event",
+      "type": "object",
+      "properties": {
+        "nodePoolName": {
+          "description": "The name of the node pool in the ManagedCluster resource",
+          "type": "string"
+        },
+        "progressState": {
```

Comment:
Please make this an enum. Here is an example of how to do this - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L497-L508

Reply:
thanks, think I updated it correctly